features/shopify-sections.md

```diff
@@ -0,0 +1,47 @@
+# Shopify Sections
+
+* [Shopify sections](shopify-sections.md#shopify-sections)
+    * [Introduction](shopify-sections.md#introduction)
+    * [How it works](shopify-sections.md#how-it-works)
+    * [Add new placement](shopify-sections.md#add-new-placements)
+
+
+
+## Introduction
+Nosto supports Shopify sections. You can add new placements through sections
+or drag-and-drop existing ones.
+
+
+```
+Warning!
```

Comment:
It'd be good to also warn users that this is in beta mode and they should reach out to support if they want to use it.